server/app/services/MessageKey.java

```diff
@@ -175,7 +175,8 @@ public enum MessageKey {
   TOAST_PROGRAM_COMPLETED("toast.programCompleted"),
   TOAST_SESSION_ENDED("toast.sessionEnded"),
   USER_NAME("header.userName"),
-  VALIDATION_REQUIRED("validation.isRequired");
+  VALIDATION_REQUIRED("validation.isRequired"),
+  TOAST_ERROR_MSG_OUTLINE("toast.errorMessageOutline");
```

Comment:
Please move this up so it is in alphabetical order with the other TOAST_ messages